cmd/crates/soroban-spec-typescript/src/boilerplate.rs

```diff
@@ -125,6 +125,10 @@ mod test {
         Ok(p)
     }
 
+    // TODO : fix the test below :
```

Comment:
should this mention addressing the dependency on futurenet also, the pr's `Why` refers to this as the main reason test is not repeatable?